scripts/collect-node-inventory.sh

```
@@ -0,0 +1,29 @@
+
+date -u +%Y-%m-%dT%H:%M:%SZ > "$out/collected_at_utc.txt"
+hostnamectl 2>/dev/null > "$out/hostnamectl.txt" || hostname > "$out/hostname.txt"
+cat /etc/os-release > "$out/os-release.txt"
+uname -a > "$out/uname.txt"
+lscpu > "$out/lscpu.txt"
+free -h > "$out/memory.txt"
+lsblk -J -o NAME,MODEL,SIZE,TYPE,FSTYPE,MOUNTPOINTS > "$out/lsblk.json"
+
+if command -v lshw >/dev/null 2>&1; then
+  if lshw -json > "$out/lshw.json" 2>/dev/null; then :; else
+    sudo -n lshw -json > "$out/lshw.json" 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)